SparkPost/matchbox
# FE-1002 Cleanup Storybook Containers

### What Changed
- Removes all individual story decorators
- Adds a single global storybook decorator with `ThemeProvider` and padding

### How To Test or Verify
- Run storybook with `npm run start:storybook`
- Verify all stories work

### PR Checklist
